Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1282,32 +1225,32 @@ private void HandleFalloffPerItem()
 
                     if (scrollDirection == ScrollDirectionType.UpAndDown)
                     {
-                        if (scrollContainer.transform.localPosition.y > maxY + (frontPlaneDistance * bounceMultiplier)
-                            || scrollContainer.transform.localPosition.y < minY - (frontPlaneDistance * bounceMultiplier))
+                        if (ScrollContainer.transform.localPosition.y > MaxY
+                            || ScrollContainer.transform.localPosition.y < minY)
                         {
-                            velocityState = VelocityState.Bouncing;
+                            CurrentVelocityState = VelocityState.Bouncing;
                             velocitySnapshot = 0.0f;
                             break;
                         }
                         else
                         {
-                            workingScrollerPos = Solver.SmoothTo(scrollContainer.transform.localPosition, velocityDestinationPos, Time.deltaTime, 0.9275f);
+                            workingScrollerPos = Solver.SmoothTo(ScrollContainer.transform.localPosition, velocityDestinationPos, Time.deltaTime, 0.9275f);
 
                             SnapVelocityFinish();
                         }
                     }
                     else
                     {
-                        if (scrollContainer.transform.localPosition.x > maxX + (frontPlaneDistance * bounceMultiplier)
-                            || scrollContainer.transform.localPosition.x < minX - (frontPlaneDistance * bounceMultiplier))
+                        if (ScrollContainer.transform.localPosition.x > maxX + (frontTouchDistance * bounceMultiplier)
```

Comment:
I wonder if someday we will use the elastics system for this ;-)

Reply:
we could try adding elasticsmanager into scrollingobject collection - Should we add an issue so someone can have a look for after graduating? 

---
In reply to: [476935788](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8380#discussion_r476935788) [](ancestors = 476935788)